README.md

```diff
@@ -137,5 +139,5 @@ Run it with:
 
 ## Conclusion
 
-Using Docker Hub to build your images each time the code is modified and pushed to repository has some limitations. As of today it will build your images maximum every 5 minutes. But it is certainly a very cheap way of doing so and should get most developers integrating good practices very early. Later on, you might want to look into hosting your own private docker registry, using base images of your making or even have your own building and pushing process for your images.
-Next time, we will explore how to run unit tests using on our application.
+As I mentioned earlier, using Docker Hub to build your images each time the code is modified and pushed to repository has some limitations. As of today it will build your images at the most every 5 minutes. But it is certainly a very cheap way of doing so [Compared to what?]. Later on, you might want to look into hosting your own private Docker registry using base images of your making. Or you may even choose to have your own building and pushing process for your images.
+Next time, we will explore how to run unit tests using [Using what?] on our application.
```

Comment:
Yep I had not decided yet between Travis and Jenkins,. I need to run a few more test with Travis (for which I'm tending too) to be sure it's doing exactly what I want.